src/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -39,6 +39,8 @@ public static partial class Http
             public static string SelfSignedCertRemoteServer => GetValue("COREFX_HTTPHOST_SELFSIGNEDCERT", "https://self-signed.badssl.com/");
             public static string RevokedCertRemoteServer => GetValue("COREFX_HTTPHOST_REVOKEDCERT", "https://revoked.grc.com/");
 
+            public static string EchoClientCertificateRemoteServer => GetValue("COREFX_HTTPHOST_ECHOCLIENTCERT", "https://corefx-net-tls.azurewebsites.net/EchoClientCertificate.ashx");
```

Comment:
Where is the code for this corefx-net-tls server? Also, why do we need yet another server name?